package/libs/readline/Makefile

```diff
@@ -8,12 +8,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=readline
-PKG_VERSION:=8.2
-PKG_RELEASE:=2
+PKG_VERSION:=8.2.13
```

Comment:
I would prefer if we stay at version 8.2, then these hacks around `PKG_VERSION` are not needed

Reply:
Makes sense, done!